_data/tables/transform-high-performing-org.yml

```diff
@@ -67,22 +110,27 @@
       data:
         list: false
         fr: 
-        en: Develop a self assessement tool based on capabilities of high performing organizations and PwC recommendations
+        en: Establish communication plan and change management for continuous improvement in (and around) IITB
     - col:
       data:
         list: false
         fr: 
-        en: 
+        en: Communicating the intended outcome to IITB employees will be very important for this transformation to succeed.
```

Comment:
Communicating the intended outcome to IITB employees will be very important for this transformation to succeed.
==>
Communicate the continous improvement framework to all employees within IITB